network-api/networkapi/wagtailpages/pagemodels/research_hub/library_page.py

```diff
@@ -33,6 +33,8 @@ class ResearchLibraryPage(BasePage):
 
     template = "pages/research_hub/library_page.html"
 
+    SORT_CHOICES = constants.SORT_CHOICES
```

Comment:
It looks like the reason the library page template was not rendering the sort choices was because in this commit https://github.com/MozillaFoundation/foundation.mozilla.org/commit/81edc5d65f1c70ea684a642d069c6379f077c997 we moved all of the sort choices to a `constants.py` file